gabrielaEnache/src/main/java/collections/utils/Generator.java

```diff
@@ -0,0 +1,9 @@
+package collections.utils;
+
+import java.util.Random;
+
+public class Generator {
+    public static long generateId() {
+        return new Random().nextInt(1000);
```

Comment:
Using random it's a good idea, however, in case we will have more than 1000 items or orders, then the ids will be not unique. I would purpose another approach : Within the Order class, create a static variable named idCOunter, then within the constructor you are going to provide this variable as id, after that it can be incremented:
private static long idCounter;
........constructor(){
this.id = idCounter++;
}

Reply:
Fixed